src/test/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializerTest.java

```diff
@@ -136,11 +160,31 @@ public void shouldSerializeAmountAsQuotedDecimal() throws JsonProcessingExceptio
     }
 
     @Test
+    public void shouldSerializeAmountAsQuotedDecimalWithDefaultFractionDigits() throws JsonProcessingException {
+        final ObjectMapper unit = unit(module().withQuotedDecimalNumbers());
+
+        final String expected = "{\"amount\":\"100.00\",\"currency\":\"EUR\"}";
+        final String actual = unit.writeValueAsString(hundred);
+
+        assertThat(actual, is(expected));
+    }
+
+    @Test
+    public void shouldSerializeAmountAsQuotedDecimalWithHigherNumberOfFractionDigits() throws JsonProcessingException {
+        final ObjectMapper unit = unit(module().withQuotedDecimalNumbers());
+
+        final String expected = "{\"amount\":\"0.0001\",\"currency\":\"EUR\"}";
+        final String actual = unit.writeValueAsString(fractions);
+
+        assertThat(actual, is(expected));
+    }
+
+    @Test
     public void shouldSerializeAmountAsQuotedDecimalPlainString() throws JsonProcessingException {
         final ObjectMapper unit = unit(module().withQuotedDecimalNumbers());
         unit.enable(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN);
 
-        final String expected = "{\"amount\":\"100\",\"currency\":\"EUR\"}";
+        final String expected = "{\"amount\":\"100.00\",\"currency\":\"EUR\"}";
```

Comment:
That is not what I would have expected and tried to address with https://github.com/zalando/jackson-datatype-money/pull/66#pullrequestreview-6. I would expect here `100.00` as result.

Reply:
Isn't that what it says?

Reply:
Sorry, is correct.